paltas/MainDeflector/perturbers.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+"""
+Provide the implementation of a perturber that consists
+of two or fewer components.
+
+This module contains the classes to render perturbers consisting of simple
+combinations of lenstronomy profiles.
+"""
+from .main_deflector_base import MainDeflectorBase
+from lenstronomy.LensModel.profile_list_base import lens_class
+
+
+class Perturber(MainDeflectorBase):
+        """Class for rendering a perturber that includes a PEMD profile, an external shear and SIS model.
+
+	Args:
+		main_deflector_parameters (dict): A dictionary containing the type of
+			main deflector and the value for each of its parameters.
+		cosmology_parameters (str,dict, or colossus.cosmology.Cosmology):
+			Either a name of colossus cosmology, a dict with 'cosmology name':
+			name of colossus cosmology, an instance of colussus cosmology, or a
+			dict with H0 and Om0 ( other parameters will be set to defaults).
+	Notes:
+		Uses the lenstronomy EPL class, which is equivalent to PEMD but is pure
+		python.
+
+	Required Parameters
+
+	- gamma - power law slope
+	- theta_E - Einstein radius of the profile in units of arcseconds
+	- e1 - x-direction ellipticity eccentricity
+	- e2 - xy-direction ellipticity eccentricity
+	- center_x - x-coordinate lens center in units of arcseconds
+	- center_y - y-coordinate lens center in units of arcseconds
+	- gamma1 - x-direction shear
+	- gamma2 - xy-direction shear
+	- ra_0 - ra origin of shear in units of arcseconds
+	- dec_0 - dec origin of shear in units of arcseconds
+	- z_lens - perturber redshift
+	"""
+	# Define the parameters we expect to find for the DG_19 model
+	required_parameters = ('gamma','theta_E','e1','e2','center_x',
```

Comment:
Need to update required parameters. You'll need to feed in new parameters for the SIS model.